django_setup_configuration/base.py

```diff
@@ -0,0 +1,174 @@
+from dataclasses import dataclass
```

Comment:
I've just noticed that the file is called `base`. I find it confusing. Could you rename it into something more specific please?